cvat-ui/src/components/header/settings-modal/styles.scss

```diff
@@ -95,12 +96,20 @@
     }
 }
 
-.cvat-settings-page-back-button {
-    width: 100px;
-    margin-top: 15px;
+.cvat-player-settings-image-preview {
+    width: 100%;
+    max-height: 180px;
+    object-fit: cover;
 }
 
-.cvat-settings-page-back-button-wrapper {
-    display: flex;
-    justify-content: flex-end;
+.canvas-background-color-picker-popover .ant-popover-inner-content {
+    padding: 6px 12px;
+
+    :first-child:first-child {
+        box-shadow: unset !important;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/cvat/cvat/pullRequest?prid=5722655)